src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -332,6 +338,17 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         dict_api_sc->reset_iterators();
     }
 
+    template <typename T>
+    void debug_emit_loc(const T &x) {
+        // TODO: convert the linear location to line and column
```

Comment:
I would make this configurable with `--debug-store-line-column`. If off (default) it would just store the linear location, which is faster. If on, it would convert to line+column, which is slower, but other debugging tools could then work with it.

Reply:
@Thirumalai-Shaktivel I thought about this and I think the best way is to just store the linear location information (as we have it in ASR) directly. It will not be directly usable in other debuggers, but it is the cleanest and most general solution, also the fastest. Will be easy to extract it from `llvm-dwarf`. We can also use the same approach with the WASM backend.

Reply:
Perfect! But I think let's keep the TODO: if needed we can implement in the future